### ISSUE TITLE
Right scroll bar is not visible on Chrome

### ISSUE THREAD

**Issue by elliotolds** [Roam-Research/issues#85]
When pages are longer than can fit on the screen, browsers display a scrollbar along the right edge of the window.

On Chrome (tested both on Window 10 and Linux Mint), the scrollbar is not visible. On Firefox it is visible as expected.

This is bad because it makes it very hard to orient yourself within the page. You can't easily see how long the page is and where you are within it.

**To Reproduce**

1. Open your Roam DB in Chrome.
2. Either find an existing long page, or make one bit repeatedly hitting enter.
3. Look on the right edge of the browser window and note that there's no scrollbar.

**System Information:**
 - Device: Desktop
 - OS: Both Windows 10 and Linux Mint 19.1
 - Browser: Chrome (verified it works on Firefox)

**Photos**

Scrollbar appearing in Firefox:
![image](https://user-images.githubusercontent.com/5849793/80928067-e9f3e300-8d56-11ea-9e9b-1342d443e56e.png)

Scrollbar missing in Chrome:
![image](https://user-images.githubusercontent.com/5849793/80928106-3dfec780-8d57-11ea-91c0-de768da5f227.png)

**Comment by rockethubble** [comment]
Yes I am also affected by this bug.

**Comment by antonylewis** [comment]
A workaround is to add user styles (I can't remember where I got this from, apologies original author):

{
    div::-webkit-scrollbar {
        width: 10px;
    }
    div::-webkit-scrollbar-track {
        background-color: #9993 !important;
    }
    div::-webkit-scrollbar-thumb {
        background-color: #9999 !important;
    }

**Comment by jbnv** [comment]
I'm observing this as well.

**Comment by ACL0815** [comment]
Workaround

An an Browser Plugin like: User JavaScript and CSS

    div::-webkit-scrollbar {
        width: 10px;
    }
    div::-webkit-scrollbar-track {
        background-color: #9993 !important;
    }
    div::-webkit-scrollbar-thumb {
        background-color: #9999 !important;
    }

**Comment by hanct** [comment]
>  div::-webkit-scrollbar {
>         width: 10px;
>     }
>     div::-webkit-scrollbar-track {
>         background-color: #9993 !important;
>     }
>     div::-webkit-scrollbar-thumb {
>         background-color: #9999 !important;
>     }

But where and how to add the user styles?

**Comment by holdenlee** [comment]
This is also true for Microsoft Edge. (I'm using version 87.0.664.52)

**Comment by hanct** [comment]
> This is also true for Microsoft Edge. (I'm using version 87.0.664.52)

Agree. Quite surprise that this "bug" still exist till now, given that this issue has been raised in May 2020..unless it is an intentional omission. :)
